### DISCUSSION TITLE
mac 编译不过问题

### DISCUSSION THREAD

**Discussion by fdgggy** [ejoy/ant#83] [Q&A]
luamake
[1/446] Compile C++ build/macos/debug/obj/vfs/vfs.obj
FAILED: build/macos/debug/obj/vfs/vfs.obj
clang -MMD -MT build/macos/debug/obj/vfs/vfs.obj -MF build/macos/debug/obj/vfs/vfs.obj.d -std=c++20  -Wall -g -Ibuild/macos/debug/lua54 -o build/macos/debug/obj/vfs/vfs.obj -c clibs/vfs/vfs.cpp
error: invalid value 'c++20' in '-std=c++20'

**Comment by actboy168** [comment]
升级你的clang
